firmware/util/loggingcentral.cpp

```diff
@@ -210,6 +208,13 @@ void efiPrintfInternal(const char *format, ...) {
 	// Ensure that the string is comma-terminated in case it overflowed
 	lineBuffer->buffer[sizeof(lineBuffer->buffer) - 1] = LOG_DELIMITER[0];
 
+	for (unsigned int i = 0; i < strlen(lineBuffer->buffer); i++) {
```

Comment:
we know the length of the written buffer, right? why do we need `strlen` again?

Reply:
Done!